ocpp/v20/schemas/PublishFirmwareStatusNotificationRequest_v1p0.json

```diff
@@ -0,0 +1,36 @@
+﻿{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "urn:OCPP:Cp:2:2018:4:PublishFirmwareStatusNotificationRequest",
+  "comment": "OCPP 2.0 - v1p0",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "status": {
+      "type": "string",
+      "additionalProperties": true,
+      "enum": [
+        "Downloaded",
+        "DownloadFailed",
+        "Downloading",
+        "DownloadScheduled",
+        "DownloadPaused",
+        "PublishFailed",
+        "Published",
+        "InvalidChecksum",
+        "ChecksumVerified"
+      ]
+    },
+    "location": {
+      "type": "array",

```

Comment:
according to the spec section 1.43.2 page 315, the PublishFirmwareStatusNotification attribute "location" has a string type. Therefore, this schema is wrong and hast to be changed to type: string with a maxlength of 512

Reply:
Good catch!